tests/wpunit/Tribe/JSON_LD/AbstractTest.php

```diff
@@ -0,0 +1,176 @@
+<?php
+namespace Tribe\JSON_LD;
+
+require_once codecept_data_dir( 'classes/Tribe__JSON_LD__Test_Class.php' );
+
+use Tribe__JSON_LD__Test_Class as Jsonld;
+
+class AbstractTest extends \Codeception\TestCase\WPTestCase {
```

Comment:
We have this format throughout all of the test code, Codeception will generate class names in he `Whatever_Class_Name__IsTest` format. To have it like that we should update all of the `codeception.yml` and tests to stick.